domains/Discussions/Policies/AnswerPolicy.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+
+namespace Domains\Discussions\Policies;
+
+
+use Domains\Accounts\Models\User;
+use Domains\Discussions\Models\Answer;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class AnswerPolicy
+{
+    use HandlesAuthorization;
+
+    public function update(User $user, Answer $answer): bool
+    {
+        return $answer->author->is($user);
```

Comment:
We could take advantage of the question we're loading in the controller, pass it to the policy and make sure that it's the one related to the answer we're trying to update, forbidding the operation otherwise.